nuxt.config.ts

```diff
@@ -29,6 +29,33 @@ export default defineNuxtConfig({
       }
     }
   },
+  runtimeConfig: {
+    public: {
```

Comment:
Consider just reading all from .env without default

Reply:
Moved to .env